### PR TITLE
Breaking: Navigation Button API compatibility (fixes #173)

### LINKED ISSUE #173: Navigation Button API compatibility
### Subject of the issue
Style and code changes are necessary to attain compatibility with Navigation Button text labels introduced in https://github.com/adaptlearning/adapt-contrib-core/issues/338

### Your environment
* Framework 5.28.9
* Page Level Progress 6.3.1
* Core https://github.com/adaptlearning/adapt-contrib-core/pull/339

### PR DESCRIPTION
**Pull request by swashbuck**
Fix #173 

### Fix
* Normalizes the nav button padding
* Adds an indicator container and sets it to the same height as the other nav button icons. This should ideally keep the indicator and text aligned with other nav buttons.

### Breaking
* Uses new Navigation Button API for the nav button

### Testing
1. `adapt-contrib-core`: Switch to the `issue/338` branch
2. In `course.json`, enable nav text labels:

```
  "_navigation": {
    "_showLabel": true
  },
```

[//]: # (Mention any other dependencies)
Dependent on https://github.com/adaptlearning/adapt-contrib-core/pull/339 for the new text label functionality.

### CODE REVIEW

File: templates/pageLevelProgressNavigation.hbs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {{! make the _globals object in course.json available to this template}}
 {{import_globals}}
 
+<div class="pagelevelprogress__indicator-wrapper"></div>
+
 <span class="tooltip">
```
Comment (oliverfoster) [inline templates/pageLevelProgressNavigation.hbs:6]: Is it worth removing the tooltip span at this point?

As far as I can see, _ariaLabels.progress label doesn't exist.
https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/schema/course.schema.json

https://github.com/adaptlearning/adapt-contrib-core/blob/3668a15c1e9777c23615746cd1d8465f9d729a27/schema/course.schema.json#L142-L300

Comment (swashbuck) [reply templates/pageLevelProgressNavigation.hbs]: Yep, agreed. Done.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by oliverfoster — APPROVED** [review]

**Comment by swashbuck** [comment]
@oliverfoster I figure we can include the JS changes in this branch, too. I could use your help, though 😊

**Comment by oliverfoster** [comment]
Sounds good.

**Review by eleanor-heath — APPROVED** [review]

**Pull request merged**